### PR TITLE
Look for RPM packages whose names look like JBoss packages

### LINKED ISSUE #341: Check installed RPMs in EAP scan
Part of #335 .

### PR DESCRIPTION
**Pull request by noahl**
Yet another lightweight way to check for JBoss installations.

Closes #341 .

### CODE REVIEW

File: roles/jboss_eap/tasks/main.yml [hunk 1]
```diff
@@ -23,3 +23,8 @@
       register: jboss.eap.processes
       ignore_errors: yes
       when: '"jboss.eap.processes" in facts_to_collect'
+    - name: check for jboss packages
+      raw: rpm -qa --qf "%{NAME}|%{VERSION}|%{RELEASE}|%{INSTALLTIME}|%{VENDOR}|%{BUILDTIME}|%{BUILDHOST}|%{SOURCERPM}|%{LICENSE}|%{PACKAGER}|%{INSTALLTIME:date}|%{BUILDTIME:date}\n" | grep -E '(eap7)|(jbossas)' | sort
```
Comment (chambridge) [inline roles/jboss_eap/tasks/main.yml:27]: Based on your middleware EAP fingerprinting section are we capturing all versions here?
(eap6)?

**Review by chambridge — APPROVED** [review]
Minor comment on completeness

File: roles/jboss_eap/tasks/main.yml (hunk 1 above)
Comment (noahl) [reply roles/jboss_eap/tasks/main.yml:27]: EAP6 used "jbossas" as its package prefix, oddly.

Comment (noahl) [reply roles/jboss_eap/tasks/main.yml:27]: And to be clear, jbossas is there only to capture EAP 6. The EAP 7 packages all have "eap7" in their name.

**Pull request merged**